index.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Open Containers
+nav_order: 1
+permalink: /
+---
+
+# The Open Container Initiative
+
+Welcome to the Open Containers Initiative documentation site! The Open Container 
```

Comment:
I'd put a parenthetical `(OCI)` after first use here and use `OCI` more often (for brevity).

Reply:
Fixed,

Reply:
For some places where a substitute of OCI could be used, since these are generally introductory docs we want to assume the reader is still learning what OCI means, and the repetition, although redundant, will help to solidify that knowledge.